ipv-stub/scripts/encrypt-message-locally.mjs

```diff
@@ -12,22 +14,29 @@ const publicKeyPem = `-----BEGIN PUBLIC KEY-----
   6wIDAQAB
   -----END PUBLIC KEY-----`;
 
-const base64Encode = (s) => {
-  return Buffer.from(s, "utf-8").toString("base64");
-};
+const authPrivateSigningKeyEVCS = `-----BEGIN PRIVATE KEY-----
+MIGHAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBG0wawIBAQQgksszURcCxE4v8xSA
+O9uwvvKDnntEb+2OBxQnsPs7vfKhRANCAAQnVd6isHfIQ7MlVbiy0wjl0gERdnca
+j0qCr6EzRoVnxYW0/4WJVr0Pz5kd2wJkSVPsX/vKDEanPgh7XmH+rehn
+-----END PRIVATE KEY-----
+`;
 
-const createJwt = (header, payload, signature) => {
-  return [header, payload, signature]
-    .map((e) => base64Encode(JSON.stringify(e)))
-    .join(".");
-};
+const authPrivateSigningKeyIPV = `-----BEGIN PRIVATE KEY-----
+MIGHAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBG0wawIBAQQgSaOnCpAfj31OwM9+
+IPuc+xPQZ6iCJHP+c3n4gOof+kihRANCAASZgtxRT+cjvTXQvGCl6Kst6k5m95C8
+E66Lggy4GZsCn3tNfuUpbdbSeBRdiNs2J1wif/VGcj+6o/RoTa+IzP3C
+-----END PRIVATE KEY-----
+`;
```

Comment:
May be worth adding a comment here to these two here indicating these are just for local use too (as to settle any security alerts before they escalate)